wagtail/actions/copy_page.py

```diff
@@ -60,6 +60,14 @@ def __init__(
         self.process_child_object = process_child_object
         self.log_action = log_action
         self.reset_translation_key = reset_translation_key
+        self._uuid_mapping = {}
+
+    def get_new_translation_key(self, old_uuid):
+        """Map old UUIDs to new UUID values."""
+        if old_uuid not in self._uuid_mapping:
+            self._uuid_mapping[old_uuid] = str(uuid.uuid4())
```

Comment:
question: is the conversion to string necessary? given we used to assign `uuid.uuid4()` before?